exercises/isogram/isogram_test.exs

```diff
@@ -0,0 +1,54 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("isogram.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule IsogramTest do
+  use ExUnit.Case
+
+  test "isogram in lowercase" do
+    refute Isogram.isogram?("eleven")
+  end
+
+  @tag :pending
+  test "not isogram in lowercase" do
+    assert Isogram.isogram?("subdermatoglyphic")
+  end
+
+  @tag :pending
+  test "not isogram in upper and lower case" do
+    refute Isogram.isogram?("Alphabet")
+  end
+
+  @tag :pending
+  test "isogram with dash" do
+    assert Isogram.isogram?("thumbscrew-japingly")
```

Comment:
If this test is driving out ignoring dashes then should the test case have multiple dashes?